mandatory/1-fix-functions.js

```diff
@@ -2,27 +2,30 @@
 
   Aim: to understand the change code inside functions 
 
-  See the below functions. They are syntactically correct but are not outputting the right results.
+  See the below functions. They are syntactically correct but are not 
+  outputting the right results.
 
   Run the tests and see how you can fix them.
 
   NOTE:Only make edits inside the function
 
 */
 
-function mood() {
-  let isHappy = true;
+function mood(isHappy) {
+  //let isHappy = true;
+  
+//  if (isHappy) {
+//    return "I am happy";
+//  } else {
+//    return "I am not happy";
+//  }
 
-  if (isHappy) {
-    return "I am happy";
-  } else {
-    return "I am not happy";
-  }
+  return isHappy ? "I am happy" : "I am not happy";
```

Comment:
This work, but you could have also just erased the line `let isHappy = true;`. The idea of this exercise is to be sure to not change a variable inside a function, as long as it is not what you want to do.